### ISSUE TITLE
Plotting Exception

### ISSUE THREAD

**Issue by teddyrendahl** [slaclab/pydm#411]
Getting the following exception with `PyDMTImePlot` on master branch. This does not occur at `v1.4.2`. Any hints?
```
  File "/Users/trendahl/miniconda3/envs/everything/lib/python3.6/site-packages/pyqtgraph-0.10.0-py3.6.egg/pyqtgraph/graphicsItems/PlotDataItem.py", line 671, in viewRangeChanged
    self.updateItems()
  File "/Users/trendahl/miniconda3/envs/everything/lib/python3.6/site-packages/pyqtgraph-0.10.0-py3.6.egg/pyqtgraph/graphicsItems/PlotDataItem.py", line 483, in updateItems
    x,y = self.getData()
  File "/Users/trendahl/miniconda3/envs/everything/lib/python3.6/site-packages/pyqtgraph-0.10.0-py3.6.egg/pyqtgraph/graphicsItems/PlotDataItem.py", line 556, in getData
    ds = int(max(1, int((x1-x0) / (width*self.opts['autoDownsampleFactor']))))
```

**Comment by hhslepicka** [comment]
@teddyrendahl i am fixing this with a PR in a couple of minutes...
It is related to this: https://github.com/pyqtgraph/pyqtgraph/pull/763

**Comment by teddyrendahl** [comment]
Oops. Didn't see that! 😄

**Comment by teddyrendahl** [comment]
Closed by #412
